devtools/ci/integration-test.sh

```diff
@@ -12,10 +12,10 @@ if [ -d "$GODWOKEN_DIR" ]
 then
     echo "godwoken project already exists"
 else
-    git clone --depth=66 -b master https://github.com/nervosnetwork/godwoken.git $GODWOKEN_DIR
+    git clone -b generator-ahead-of-time-compilation https://github.com/flouse/godwoken.git $GODWOKEN_DIR
```

Comment:
use nervosnetwork repo? we already merge the PR into develop

Reply:
Done